modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc

```diff
@@ -33,10 +33,7 @@ image::mobile-clients-service-bind-3.png[]
 NOTE: You can safely close the dialog, the process continues in the background.
 
 After the binding is complete, the configuration in `mobile-services.json` for the {mobile-client} is updated in the
-*Configuration* tab.
-This configuration is used by your app to interact with the bound services.
-
-image::mobile-clients-overview-mobile-services-json.png[]
```

Comment:
Hi @finp , following on from our discussion at the stand-up this morning, I reviewed the text that relates to the image and I think that the image isn't adding value and as such, I've removed it. If you think it's best to remove this image too, I'll go ahead and delete the actual png file and merge. Can you let me know what you think?

Reply:
@jstaffor  go ahead and delete image.

Reply:
:+1: